docs/legal/dpa/index.rst

```diff
@@ -0,0 +1,406 @@
+Data Processing Agreement
+=========================
+
+This Data Processing Agreement (“\ **DPA**\ ”) is an addendum to the
+Terms of Service (“\ **Agreement**\ ”) between Read the Docs, Inc.,
+along with our affiliates and subsidiaries (collectively, “\ **Read the
+Docs**,” “\ **us**,” or “\ **we**\ ”) and the organization subscribing
+to our Services (“\ **Organization**\ ”). This DPA takes effect on the
+date Organization signs up for Services, and governs the collection,
```

Comment:
Should we make clear this is for RTD for Biz?

Reply:
Good question. Maybe a call out at the top of the page, outside the legal copy?

Reply:
I went ahead and added an admonition to the top of the page for now.  This will be a specific RTD for Business agreement for now.